src/api/pokemon.spec.js

```diff
@@ -0,0 +1,17 @@
+
+// example result of calling getPokemonById(1):
+// { id: 1, name: 'Bulbasaur', photoUrl: 'someurl', types: [ { id: 3, name: 'grass' }, { id: 6, name: 'poison' } ] }
+// example result of calling getTypeById(3):
+// { id: 3, name: 'grass' }
+
+import { getPokemonById, getTypeById } from "../api/pokemons.js";
+
+describe("Get pokemon details from API", function(){
+    test("Should return pokemon details based on its id.", function() {
+        expect(getPokemonById(2)).toEqual("Surprise 2")
```

Comment:
tests should be written with formula: "given, when, then" for example (inside of test):
```
//given
const firstNumber = 2;
const secondNumber = 5;

//when
const result = add(firstNumber, secondNumber);

//then
expect(result).toEqual(7)
```

Reply:
It will make them more readable for other people :) and will also make you think what is your "input", what is the behaviour/function you are testing and what you expect to happen then